tests/SprykerEcoTest/Yves/CrefoPay/Quote/CrefoPayQuoteExpanderTest.php

```diff
@@ -0,0 +1,154 @@
+        $clientMock->expects($this->once())->method('startCrefoPayTransaction');
+        $crefoPayQuoteExpander = $this->createCrefoPayQuoteExpander($clientMock);
+
+        $quoteTransfer = (new QuoteTransfer())->setCrefoPayTransaction(new CrefoPayTransactionTransfer());
+
+        // Act
+        $crefoPayQuoteExpander->expand($this->tester->getRequest(), $quoteTransfer);
+    }
+
+    /**
+     * @return void
+     */
+    public function testExpandShouldExecuteClientCallWhenCountriesAreNotProvided(): void
+    {
+        // Arrange
+        $clientMock = $this->createClientMock();
+        $clientMock->expects($this->once())->method('startCrefoPayTransaction');
+        $crefoPayQuoteExpander = $this->createCrefoPayQuoteExpander($clientMock);
+
+        $quoteTransfer = (new QuoteTransfer())
+            ->setCrefoPayTransaction((new CrefoPayTransactionTransfer())->setIsSuccess(true));
+
+        // Act
+        $crefoPayQuoteExpander->expand($this->tester->getRequest(), $quoteTransfer);
+    }
+
+    /**
+     * @return void
+     */
+    public function testExpandShouldExecuteClientCallWhenCountriesAreNotEqual(): void
+    {
+        // Arrange
+        $clientMock = $this->createClientMock();
+        $clientMock->expects($this->once())->method('startCrefoPayTransaction');
+        $crefoPayQuoteExpander = $this->createCrefoPayQuoteExpander($clientMock);
+
+        $crefoPayTransactionTransfer = (new CrefoPayTransactionTransfer())
+            ->setIsSuccess(true)
+            ->setBillingAddress((new CrefoPayApiAddressTransfer())->setCountry(static::COUNTRY_CODE_DE));
+
+        $quoteTransfer = (new QuoteTransfer())
+            ->setCrefoPayTransaction($crefoPayTransactionTransfer)
+            ->setBillingAddress((new AddressTransfer())->setIso2Code(static::COUNTRY_CODE_US));
+
+        // Act
+        $crefoPayQuoteExpander->expand($this->tester->getRequest(), $quoteTransfer);
+    }
+
+    /**
+     * @return void
+     */
+    public function testExpandShouldNotExecuteClientCallWhenCountriesAreEqual(): void
+    {
+        // Arrange
+        $clientMock = $this->createClientMock();
+        $clientMock->expects($this->never())->method('startCrefoPayTransaction');
+        $crefoPayQuoteExpander = $this->createCrefoPayQuoteExpander($clientMock);
+
+        $crefoPayTransactionTransfer = (new CrefoPayTransactionTransfer())
+            ->setIsSuccess(true)
+            ->setBillingAddress((new CrefoPayApiAddressTransfer())->setCountry(static::COUNTRY_CODE_DE));
+
+        $quoteTransfer = (new QuoteTransfer())
+            ->setCrefoPayTransaction($crefoPayTransactionTransfer)
+            ->setBillingAddress((new AddressTransfer())->setIso2Code(static::COUNTRY_CODE_DE));
+
+        // Act
+        $crefoPayQuoteExpander->expand($this->tester->getRequest(), $quoteTransfer);
+    }
+
+    /**
+     * @return \PHPUnit\Framework\MockObject\MockObject|\SprykerEco\Client\CrefoPay\CrefoPayClientInterface
```

Comment:
It can be moved to tester

Reply:
I think it should be in the test since this is test specific function.